CODE_OF_CONDUCT.md

```diff
@@ -1,3 +1,9 @@
-## Microsoft Open Source Code of Conduct
+# Microsoft Open Source Code of Conduct
```

Comment:
That's too big of a heading in my opinion.  Let's keep it as `##`

Reply:
The template use to exist at a different page, most likely at https://opensource.microsoft.com/. 

Now they updated it to exist at this https://github.com/microsoft/repo-templates/blob/main/shared/CODE_OF_CONDUCT.md.

Reply:
Oh. I see you mean this from the README.md

https://github.com/microsoft/repo-templates/blob/main/README.md#code-of-conduct

Reply:
Yes. There is no guidance on updating the README atm.